MAiD - Case Management App/force-app/main/default/objects/Referral__c/validationRules/Phone_number_Validation.validationRule-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Phone_number_Validation</fullName>
+    <active>true</active>
+    <errorConditionFormula>OR(AND(Individual_Cell_Phone_Number__c!=null,OR(NOT(ISNUMBER(Individual_Cell_Phone_Number__c)),LEN(Individual_Cell_Phone_Number__c)!=10)),
```

Comment:
Salesforce autoformats phone numbers on entry from 6045211234 to (604) 521-1234 and this is why non-numerical characters are allowed. Test this after auto-formatting as you may need a REGEX that approves (999) 999-9999 which is length 14. @lilianenassar-bib